CMake/resolve_dependency_modules/template.cmake

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+include_guard(GLOBAL)
+
+if(DEFINED ENV{VELOX_<PACKAGE>_URL})
```

Comment:
Is this template being used anywhere ? It seems like we need custom files for every dependency.

Reply:
not by the cmake, this is more as form of documentation to aid in adding other dependency modules to the system.